chore: refactor project structure, add font size settings

xianghongai opened the pull request:
Need to be configured before publishing: `manifest.json` - `applications - gecko - id`

## refactor project structure

![structure](https://user-images.githubusercontent.com/2075006/56179077-097a7980-6037-11e9-9b02-75d475686b98.png)


## add font size settings

![font size](https://user-images.githubusercontent.com/2075006/56179027-e0f27f80-6036-11e9-9d71-528efd8eebaf.png)

xianghongai committed:
chore: refactor project structure, add font size settings

painyeph commented:
@xianghongai Thanks for adding new feature. It appears to be some bugs in the code. Change font size neither works in my Firefox nor Chrome. And the KaTeX rendering is broken as many symbols that should be serif becomes sans-serif. Finally, I think font size can be changed by ways like `CTRL` + `+`, instead of introducing a new panel. Setting body font size (especially in px) is not a good way.

xianghongai commented:
It was so embarrassing, I just adjusted the project structure, added a function, Firefox should be able to run well, my local test passed. Chrome is not supported because there is a storage interface provided by firefox.

The browser is a default font size of 16 pixels, all using a relative font size, is a suitable solution, when opening many documents we need to set a uniform font size.

sagarpanchal commented:
`The character encoding of the plain text document was not declared. The document will render with garbled text in some browser configurations if the document contains characters from outside the US-ASCII range. The character encoding of the file needs to be declared in the transfer protocol or file needs to use a byte order mark as an encoding signature.`


`Warning: KaTeX doesn't work in quirks mode. Make sure your website has a suitable doctype.`
